### DISCUSSION TITLE
I merged 2 pull requests without a review

### DISCUSSION THREAD

**Discussion by skanda890** [community/community#129851] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I merged 2 pull requests without a review, but did not earn the YOLO badge X2.

**Comment by PipoAT** [comment, marked as answer]
@skanda890 The YOLO badge is 100% unlocked when you merge one pull request without a review. The YOLO achievement is not one that is tiered unlike some of the other achievements. Thus, merging multiple pull requests will not have it display X2, etc. 

My answer/suggestion is based on the [official documentation](https://github-profile-achievements.github.io/english/)

**Reply by ebndev** [reply]
@skanda890 thanks for asking this and @PipoAT thanks for taking the time to answer + link documentation. Super helpful!
